ui/src/components/entity/EntityAnnotationLayer.tsx

```diff
@@ -309,6 +308,14 @@ class EntityAnnotationLayer extends React.Component<Props, {}> {
               />
             );
           } else if (isSymbol(entity)) {
+            /*
+             * Only show annotations for identifiers and functions (i.e., not operators).
+             */
+            const ANNOTATED_SYMBOL_TYPES = ["identifier", "function"];
```

Comment:
do we have a config for the UI that determines this?

Reply:
No, we do not 😞  Do you think we should have one?

Reply:
yeah probably, but it seems good to do this for now.  maybe add a TODO for migrating to config at later date if it ever becomes necessary